rotkehlchen/chain/evm/frontend_structures/types.py

```diff
@@ -0,0 +1,46 @@
+from enum import auto
+from typing import Literal, NamedTuple, Optional
+
+from rotkehlchen.utils.mixins.serializableenum import SerializableEnumMixin
+
+
+class ProtocolDetails(NamedTuple):
```

Comment:
This should be CoutnerpartyDetails, yes we had this conversation before. Kraken is also a counterparty and it's not a protocol. And it should be populated from the decoders themselves. Or ... well returned by the decoders which use them. Not all be in one big constant like you have it now.